templates/galaxy/config/object_store_conf.xml.j2

```diff
@@ -162,5 +162,16 @@ The storage consists of 16 Gen6 type A200 storage nodes and 16 Gen5 type X410 no
             <extra_dir type="job_work" path="/data/jwd/main"/>
             <extra_dir type="temp" path="/data/jwd/tmp"/>
         </backend>
+
+        <backend type="swift" id="s3_cyfronet01" weight="0" store_by="uuid">
+            <location latitude="50.0689816" longitude="19.9070188" />
+            <auth access_key="{{ s3_cyfronet01.usegalaxy_eu.access_key }}" secret_key="{{ s3_cyfronet01.usegalaxy_eu.secret_key }}" />
+            <bucket name="{{ s3_cyfronet01.usegalaxy_eu.bucket_name }}" use_reduced_redundancy="False" max_chunk_size="250" />
+            <connection host="s3p.cloud.cyfronet.pl" port="443" is_secure="True" conn_path="" multipart="True"/>
+            <cache path="/data/jwd/s3_object_store_cache" size="10000" />
+            <extra_dir type="job_work" path="/data/jwd/main"/>
```

Comment:
can/should we use the NetApp here?

Reply:
We should not be using either `jwd` or `jwd03f` here.